kevinChuang/app/template/bunny/small_bunny.html

```diff
@@ -0,0 +1,4 @@
+<div class="">
+  <h2>Thumbnail Directive</h2>
+  <img src="{{image}}" height="100" width="100" alt="{{title}}">
+</div>
```

Comment:
You pass the description property all of the way down to here but then you never actually use it.  Just be aware of that.